lib/twilio.rb

```diff
@@ -0,0 +1,19 @@
+require 'rubygems'
+# require 'twilio-ruby'
+
+class Twilio
+
+def send_text_message
+ 
+account_sid = ACd809165d63ca512f36f6d2e6846c2a0f
+auth_token = '{{ auth_token }}'
+@client = Twilio::REST::Client.new account_sid, auth_token
+ 
+message = @client.account.messages.create(:body => "Thank you! Your order was placed and will be delivered before 18:52",
```

Comment:
Metrics/LineLength: Line is too long. [121/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)